action.yml

```diff
@@ -30,8 +30,18 @@ inputs:
     required: false
 
 runs:
-  using: "docker"
-  image: "docker/Dockerfile"
+  using: 'composite'
+  steps:
+    - id: typos
+      run: $GITHUB_ACTION_PATH/docker/entrypoint.sh
```

Comment:
I feel like this file should move into an `action` folder or something

Reply:
Moved.